exdb/static/exdb/js/search_results.js

```diff
@@ -0,0 +1,8 @@
+"use strict";
+$(document).ready(function () {
+    $('#search-results').tablesorter({
+        theme: "bootstrap",
```

Comment:
This is unnecessary since we don't have any of the tablesorter CSS files. We'll add them and theme it during the UI eventual overhaul.